### DISCUSSION TITLE
How to scan a remote GitHub repository directly ?

### DISCUSSION THREAD

**Discussion by subhamku2020** [Checkmarx/kics#5916] [Q&A] [answered]
Hi Team,

After installing kics using docker option locally, how do we scan a remote repo using cli variant!

**Comment by rafaela-soares** [comment, marked as answer]
Hello @subhamku2020,

You need to use the prefix `git::`. For example: `docker run checkmarx/kics scan -p git::https://github.com/Checkmarx/kics`.

KICS provides other amazing ways of scanning, such as Terraformer (example: `docker run -e AWS_SECRET_ACCESS_KEY -e AWS_ACCESS_KEY_ID -e AWS_SESSION_TOKEN checkmarx/kics:latest scan -p "terraformer::aws:vpc:eu-west-2" -v --no-progress`) that scans resources deployed in the Cloud. 

It also provides Kuberneter (example: `docker run -v <credentials_path>:/credentials -v ${PWD}:/path/ -e K8S_CONFIG_FILE=/credentials/<config-file-name> checkmarx/kics:latest scan -p "kuberneter::*:*:*" -v --no-progress -o /path/results`) that scans resources deployed in the runtime K8s cluster. 

If you have interest, take a look at the documentation of [Terraformer](https://github.com/Checkmarx/kics/blob/master/docs/integrations_terraformer.md) and [Kuberneter](https://github.com/Checkmarx/kics/blob/master/docs/integrations_kuberneter.md).

You can also find other types of paths for the scan [here](https://docs.kics.io/latest/running-kics/).

Let me know if it helps you, please 😊
